src/portfolio-rebuild/about-page/about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Erica Renée Faulkenberry</title>
+  <link href="https://fonts.googleapis.com/css?family=Poppins:900|Roboto+Mono&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/reset.css">
+  <head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
+  </head>
```

Comment:
multiple head tags?

Reply:
don't think you need this

Reply:
whoops yeah that is a mistake -- thanks!